Resources/Regression/qml/RegressionLinearBayesian.qml

```diff
@@ -220,9 +220,26 @@ Form {
 			{
 				name: "modelPrior"
 				title: qsTr("Model Prior")
+				RadioButton { value: "uniform"; label: qsTr("Uniform"); checked: true }
 				RadioButton
 				{
-					value: "beta.binomial"; label: qsTr("Beta binomial"); checked: true
+					value: "Wilson"
+					label: qsTr("Wilson")
+					childrenOnSameRow: true
+					childrenArea.columnSpacing: 1
+					DoubleField { name: "wilsonParamLambda"; label: qsTr("λ"); defaultValue: 1; inclusive: "maxOnly"}
+				}
+				RadioButton
+				{
+					value: "Castillo"
+					label: qsTr("Castillo")
+					childrenOnSameRow: true
+					childrenArea.columnSpacing: 1
+					DoubleField { name: "castilloParamU"; label: qsTr("u"); defaultValue: 1; inclusive: "maxOnly"; min: 1}
```

Comment:
You should either specify `inclusive: yes` or set the minimum to be higher than 1